frontend/StructureViewer.vue

```diff
@@ -340,6 +341,7 @@ export default {
                 })
             })
         })
+	window.addEventListener('resize', () => this.stage.handleResize())
```

Comment:
Can you fix the indentation please and make sure the event listener is removed in beforeDestroy?